src/_nebari/stages/infrastructure/__init__.py

```diff
@@ -359,6 +359,7 @@ class GCPNodeGroup(schema.Base):
 
 
 class GoogleCloudPlatformProvider(schema.Base):
+    model_config = ConfigDict(coerce_numbers_to_str=True)
```

Comment:
It was interpreting `1.29` as a float instead of a string, so I had to make it explicitly use a string